### ISSUE TITLE
add statistical analysis of past 10 days change

### ISSUE THREAD

**Issue by actuallyaryaman** [Doofenshmirtz-Evil-Incorp/Fuelish#8]
add a bar-graph for a particular district using graph js.

**Comment by sophiya02** [comment]
Hi, Can I work on this issue??
Looking forward to hearing from you regarding any specific information you would like to share before starting.

**Comment by actuallyaryaman** [comment]
Hi Sophiya! It would be nice to have you working on our project.
Just a few steps: 

> Kindly let

**Comment by actuallyaryaman** [comment]
Hi @sophiya02, we'd love to have you working on this project.
Kindly go through and get familiar with our code base of the frontend(Vanila JS), the backend (python) and our [Contributing.md](https://github.com/Doofenshmirtz-Evil-Incorp/Fuelish#contributing) 
You can start contributing at your convenience!

Also don't forget to star this repo!

**Comment by Anjali-Kedia** [comment]
@actuallyaryaman  I would like to work on this issue

**Comment by actuallyaryaman** [comment]
Sure. You are welcome to go through the codebase and make your PR.
I suggest using the chart.js library to move forward with this.
